cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -200,50 +101,275 @@
             }
         });
     }
+
+    // Expandable Overrides
```

Comment:
This looks like it could use a `TODO` to differentiate this from an expandable. If it needs to override the expandable it's not an atomic expandable but perhaps something that both the org-chart and expandables should inherit.